test/lang/csharp/csharp-npgsql6.csproj

```diff
@@ -0,0 +1,23 @@
+<!--
+Copyright Materialize, Inc. and contributors. All rights reserved.
+
+Use of this software is governed by the Business Source License
+included in the LICENSE file.
+
+As of the Change Date specified in that file, in accordance with
+the Business Source License, use of this software will be governed
+-->
+
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="npgsql" Version="6.0.2" />
+    <PackageReference Include="NUnit" Version="3.12.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.16.1" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
+  </ItemGroup>
+</Project>
```

Comment:
It _should_ be as simple as just updating test/lang/csharp/test.sh to look like:

```bash
dotnet test csharp-npgsql5.csproj
dotnet test csharp-npgsql6.csproj
```

(This PR'll need to update that file anyway since right now it looks like it broke the reference in that file to csharp-npgsql.csproj.)

Reply:
Err, I had this in my local changes by apparently forgot to add it to the diff.